src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -90,7 +91,11 @@ def register_options(cls, register, register_jvm_tool):
     register('--suppress-output', action='store_true', default=True,
              help='Redirect test output to files in .pants.d/test/junit.')
     register('--cwd', default=_CWD_NOT_PRESENT, nargs='?',
-             help='Set the working directory. If no argument is passed, use the first target path.')
+             help='Set the working directory. If no argument is passed, use the first target path. '
+                  'If -target-root is passed, each test will use the target path of its target for '
+                  'its cwd. If test_cwd is set on a target, it will supersede this argument.')
+    register('--purge-intermediate-jvms', action='store_true', default=False,
```

Comment:
I like this feature, but let's pull this out to a different change.  Also, for the naming of the argument, what do you think of '--restart-jvms'  or something like that. Purge to me sounds like something is being deleted from the filesystem.